helm/tests/values_test.go

```diff
@@ -0,0 +1,125 @@
+package tests // nolint: testpackage
+
+import (
+	"golang.org/x/xerrors"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// LoadChart loads the chart from the parent directory.
+func LoadChart() (*WrappedChart, error) {
+	ch, err := loader.LoadDir("..")
+	if err != nil {
+		return nil, xerrors.Errorf("failed to load chart: %w", err)
+	}
+	if ch == nil {
+		return nil, xerrors.New("chart should not be nil")
+	}
+
+	originalValues, err := mapToValues(ch.Values)
+	if err != nil {
+		return nil, xerrors.Errorf("failed to convert values to map: %w", err)
+	}
+
+	return &WrappedChart{
+		Chart:          ch,
+		Metadata:       ch.Metadata,
+		Template:       ch.Templates,
+		Files:          ch.Files,
+		OriginalValues: originalValues,
+	}, nil
+}
+
+// WrappedChart is a wrapper around helm.sh/helm/v3/pkg/chart.WrappedChart.
+type WrappedChart struct {
+	Chart          *chart.Chart
+	Metadata       *chart.Metadata
+	Template       []*chart.File
+	Files          []*chart.File
+	OriginalValues *Values
+}
+
+// Values is the top-level values struct for the Coder chart.
+type Values struct {
```

Comment:
Instead of having to maintain this type like we did in v1 (and forgot to do many times) can we just use `map[string]any` as the values type for test cases instead?

Reply:
Decided to go one level further and just run `helm template` instead; it's conceptually much simpler and removes the extra k8s API deps.